src/rewriter.coffee

```diff
@@ -236,13 +241,6 @@ exports.Rewriter = class Rewriter
       #       a: b
       #       c: d
       #
-      # and
```

Comment:
this comment seems to reference an old syntax that no longer compiles (now the *first* arg has to be an implicit object in order for indentation to imply an implicit call?)